### ISSUE TITLE
TreeSheets on Windows not playing well with Google Drive

### ISSUE THREAD

**Issue by gussmith23** [aardappel/treesheets#92]
Apologies for lack of details - I will provide any more details needed; just tell me what you need.

I often (not always) get the error "Error writing TreeSheets file! (try saving under new filename)" when writing my TreeSheets file. This file is stored in a Google Drive-controlled directory on my Windows machine, and I assume that's part of the problem. (If it's not part of the problem, then this is a more serious issue!) Furthermore, after saving the file under a new filename, I can't delete the old one without elevated privilege due to permissions issues - the old file is owned by an administrator account, apparently. This is not a TreeSheets issue, I assume - it's just another indication that it may be an interaction with Google Drive that is to blame.

Steps to reproduce:

1. Create TreeSheets file in a Google Drive-controlled directory
2. Modify and save the file over and over until you get the error

If I get some free time at some point, I may try debugging this myself.

**Comment by gussmith23** [comment]
Conversation with Wouter suggested that this is likely not TS's fault, but instead, Windows or Google Drive's fault.

Wouter suggested it might be "fixed" (or just ameliorated) by TS by adding a retry dialog when saving fails. 

Feel free to close this or mark it as not a bug. I would argue that having this documented is worthwhile, but if it's not TS's fault, then it's not a bug, exactly.

**Comment by gussmith23** [comment]
Some additional details:
Sometimes -- in fact, most of the time, in my experience -- after saving fails, TS (I think it's coming from TS) will pop up a new dialog saying that the file has changed in storage, and asks whether or not you would like to reload. Hitting "yes" loads a "new" version that is exactly the same. After the file is reloaded, it can be saved like normal, without TS throwing an error.

**Comment by aardappel** [comment]
No, lets keep this bug open. Working well with cloud storage services is very important for TreeSheets, and even though it looks like Google Drive has a problem aggressively locking files (that DropBox doesn't have), it be good for TreeSheets to provide a workaround.
